.rubocop.yml

```diff
@@ -645,7 +644,7 @@ Style/BlockComments:
 Style/BlockEndNewline:
   Description: Put end statement of multiline block on its own line.
   Enabled: true
-Style/Blocks:
+Style/BlockDelimiters:
```

Comment:
@okibar, I'm curious why you changed these, was it because of updates to rubocop?

Thanks.